File: Data Structures & Algorithms/anagram-groups/submission-5.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs) {
        Map<String, List<String>> mp = new HashMap<>();

        for(String str: strs){
            char[] s = str.toCharArray();
            Arrays.sort(s);
            String sorted = new String(s);

            if(mp.containsKey(sorted)){
                mp.get(sorted).add(str);
            }else{
                mp.put(sorted, new ArrayList<>());
                mp.get(sorted).add(str);
            }
        }

        return new ArrayList<>(mp.values());
    }
}

File: Data Structures & Algorithms/is-anagram/submission-5.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        s_arr = list(s)
        t_arr = list(t)

        t_arr = sorted(t_arr)
        s_arr = sorted(s_arr)

        return(t_arr == s_arr)